README.md

```diff
@@ -405,21 +405,19 @@ await octokit.rest.issues.create({
 });
 ```
 
-The REST API endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID or parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
+The `octokit.rest` endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID and parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
 
-Every endpoint method is an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
+Under the covers, every endpoint method is actually an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
```

Comment:
Question": Does every *method* have it's own **instance** of `request`.  Feels like a bit too much implementation detail that leads to that kind of question. What's the main user-facing point you're looking to convey here?